vignettes/articles/05-data-cleaning.Rmd

```diff
@@ -358,12 +365,15 @@ ggplot(data = df.plot.ambig,
 
 Let's deal with the easy ones first. 
 
-**ambigID -337: `motusTagID`s 10811 and 16011**
+**`ambigID` -337: `motusTagID`s 10811 and 16011**
 
 ```{r}
 tbl.alltags.sub %>%
   filter(ambigID == -337) %>%
-  count(motusTagID, tagDeployStart, tagDeployEnd, tagDepLat, tagDepLon)
+  count(motusTagID, tagDeployStart, tagDeployEnd, tagDepLat, tagDepLon) %>% 
+  collect() %>% 
+  mutate(tagDeployStart = as_datetime(tagDeployStart),
+         tagDeployEnd = as_datetime(tagDeployEnd))
```

Comment:
I like your updates to use `tbl.alltags.sub` here instead of a dataframe like in the old book because it is faster/uses less memory. However, I do think we need to keep the timestamp conversions I added here because it's summarizing data pulled directly from a `tbl`, so `ts` is still numeric. There's no error if we don't convert, but the resulting summary isn't as easily interpreted by users if `ts` isn't converted to a datetime. Same for other cases where we're pulling summary information directly from `tbl`s.